trieste/acquisition/optimizer.py

```diff
@@ -429,6 +552,39 @@ def target_func_with_vectorized_inputs(
     return optimizer
 
 
+def batchify_vectorize(
+    batch_size_one_optimizer: AcquisitionOptimizer[SP],
+    batch_size: int,
+) -> AcquisitionOptimizer[SP]:
+    """
+    A wrapper around our :const:`AcquisitionOptimizer`s. This class wraps a
+    :const:`AcquisitionOptimizer` to allow it to optimize batch acquisition functions.
+
+    Unlike :meth:`batchify_joint`, :meth:`batchify_vectorize` is suitable
+    for :class:`VectorizedAcquisitionFunction`s who's individual batch element can be
```

Comment:
FYI the trailing `s` after the inlined `VectorizedAcquisitionFunction` is causing the docs failure (see https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#inline-markup-recognition-rules point 7). You could probably escape it as `\s`, but I think this is clearer:

```suggestion
    for a :class:`VectorizedAcquisitionFunction` whose individual batch element can be
```